docs/system_overview/troubleshooting.md

```diff
@@ -25,7 +199,7 @@ Some settings within iRODS can be useful when developing for iRODS or when worki
  | LOG_SYS_WARNING |  2           |
  | LOG_SYS_FATAL   |  1           |
 
-- `spLogSql=1` - This will send to the rodsLog the bind variables, the SQL query, and the return values going to and coming from the database.  This needs to be set on the iCAT server.  Setting this on a resource server will have no effect.
+- `spLogSql=1` - This will send the bind variables, the SQL query, and the return values going to and coming from the database to the log file. This needs to be set on the iCAT server. Setting this on a resource server will have no effect.
```

Comment:
iCAT server -> catalog service provider
and
resource server -> catalog service consumer